bundles/org.palladiosimulator.analyzer.slingshot.core/src/org/palladiosimulator/analyzer/slingshot/core/driver/SlingshotSimulationDriver.java

```diff
@@ -40,38 +38,34 @@ public class SlingshotSimulationDriver implements SimulationDriver {
 	private SimuComConfig config;
 
 	@Inject
-	public SlingshotSimulationDriver(final SimulationEngine engine,
-			final Injector injector,
+	public SlingshotSimulationDriver(final Injector injector, final SimulationEngine engine,
 			@SimulationBehaviorExtensions final List<SimulationBehaviorContainer> behaviorContainers) {
-		this.engine = engine;
 		this.parentInjector = injector;
 		this.behaviorContainers = behaviorContainers;
+		this.engine = engine;
 	}
 
 	@Override
 	public void init(final SimuComConfig config, final IProgressMonitor monitor) {
-		if (this.initialized) {
-			return;
-		}
-		this.monitor = monitor;
-		this.config = config;
-
-		final List<Module> partitionIncludedStream = new ArrayList<>(behaviorContainers.size() + 1);
-
-		partitionIncludedStream.add(new SimulationDriverSubModule(monitor, config));
+		final List<Module> partitionIncludedStream = new ArrayList<>(behaviorContainers.size() + 2);
```

Comment:
why did this change from +1 to +2?

Reply:
good that you noticed, iirc that's an accidental leftover from when i had split the extension points.